Makefile

```diff
@@ -1,4 +1,5 @@
 GOFLAGS :=
+DOCKER_ORG ?= $(USER)
```

Comment:
I feel like I should hardcode a conversion from `deads` to `deads2k` :)

Reply:
Sign me up for it too ;)